test/e2e/upgrade/upgrade.go

```diff
@@ -412,11 +412,15 @@ func clusterUpgrade(f *framework.Framework, c configv1client.Interface, dc dynam
 		Network  string
 	}
 	var upgradeDurationLimits = map[limitLocator]float64{
-		{configv1.AWSPlatformType, OVN}:       85,
+		// NOTE: The OVNAWSPlatformType and OVNGCPPlatformType upgrade times are originally 85 and 90 minutes each.
+		// Due to well-known reasons (https://issues.redhat.com/browse/SDN-4042), these numbers are increased to
+		// 100 minutes only for the 4.13->4.14 upgrades - same as OVNAzurePlatformType. These numbers will be brought
+		// down to their original values in OCP 4.15 (https://issues.redhat.com/browse/OTA-999)
+		{configv1.AWSPlatformType, OVN}:       100,
 		{configv1.AWSPlatformType, SDN}:       95,
 		{configv1.AzurePlatformType, OVN}:     100,
 		{configv1.AzurePlatformType, SDN}:     100,
-		{configv1.GCPPlatformType, OVN}:       90,
+		{configv1.GCPPlatformType, OVN}:       100,
 		{configv1.GCPPlatformType, SDN}:       75,
 		{configv1.BareMetalPlatformType, OVN}: 80,
```

Comment:
Have we done enough testing to know if we fit in the 80 and 95min values for BareMetalPlatformType and VSpherePlatformType respectively?

Reply:
VSphere takes https://privatebin.corp.redhat.com/?77d86d9d24448e5b#AESR21RRTdUDYEG7Hx8umJ7AySZnxavCfbwtDKG5m5hR 87minutes roughly, so we are still under the 95 minutes mark.
Awaiting the BM results...

Reply:
Apparently BM took 95 min against threashold of 80 min as discussed in other slack thread
https://privatebin.corp.redhat.com/?ada3456156ec1790#6GC2WKSkyGNysmf7QsJWhZrRzNJtDtGZEbU3Bq1QXFet

Reply:
Yea, will update both platforms - vsphere & BM for now to be safe.. anyuiw in 4.15 we should be able to totally revert this PR